src/internal/packager2/inspect.go

```diff
@@ -18,19 +18,27 @@ import (
 	"github.com/zarf-dev/zarf/src/internal/packager/kustomize"
 	"github.com/zarf-dev/zarf/src/internal/packager/template"
 	"github.com/zarf-dev/zarf/src/internal/packager2/layout"
-	layout2 "github.com/zarf-dev/zarf/src/internal/packager2/layout"
+	"github.com/zarf-dev/zarf/src/pkg/cluster"
+	"github.com/zarf-dev/zarf/src/pkg/packager/filters"
 	"github.com/zarf-dev/zarf/src/pkg/state"
 	"github.com/zarf-dev/zarf/src/pkg/utils"
 	"github.com/zarf-dev/zarf/src/pkg/variables"
+	"github.com/zarf-dev/zarf/src/pkg/zoci"
 	"helm.sh/helm/v3/pkg/chartutil"
 )
 
 type ResourceType string
+type InspectTarget string
```

Comment:
While the type itself should probably change names, maybe LayerCategory (names are hard), I feel that this type should live in zoci. It helps the code articulate that these targets are only relevant when performing a pull from an OCI target.

Reply:
Agreed - I think i've iterated in a few circles where it makes sense for these to finally reconcile in the best location.